src/main/java/io/polyrepo/analyser/util/ConnectionUtil.java

```diff
@@ -0,0 +1,26 @@
+package io.polyrepo.analyser.util;
+
+import org.apache.commons.dbcp2.BasicDataSource;
+
+import java.sql.Connection;
+import java.sql.SQLException;
+
+public class ConnectionUtil {
+
+    private static BasicDataSource basicDataSource = new BasicDataSource();
+
+    static {
+        basicDataSource.setUrl("jdbc:mysql://localhost:3306/polyrepodb");
+        basicDataSource.setUsername("root");
```

Comment:
learn about how spring boot can help with picking up different property files in different environments. Like dev, test, prod.